index.html

```diff
@@ -12,6 +12,7 @@ <h2>I love pizza!</h2>
             <li>dough</li>
             <li>sauce</li>
             <li>cheese</li>
+            <li>mushrooms</li>
```

Comment:
I am allergic to mushrooms - please swap with olives

Reply:
swapped with olives - pls approve